go/streamlog/streamlog.go

```diff
@@ -90,6 +99,9 @@ func registerStreamLogFlags(fs *pflag.FlagSet) {
 	// QueryLogFormat controls the format of the query log (either text or json)
 	fs.StringVar(&queryLogFormat, "querylog-format", queryLogFormat, "format for query logs (\"text\" or \"json\")")
 
+	// QueryLogJSONFormat controls whether the new querylog json format is used
+	fs.BoolVar(&queryLogJSONV2, "querylog-json-v2", false, "use v2 format for querylog-format=json")
```

Comment:
If we already have `--querylog-format=json`, why not `--querylog-format=json-v2` rather than a new flag?